source/getstarted.html.erb

```diff
@@ -60,13 +49,13 @@ title: Get started
               <span class="task-list-section-number">3.</span> Share with your team
             </h2>
             <div class="task-list__details">
-              <p class="govuk-body">Share information about GOV.UK&nbsp;Pay with your broader team. Managing payments
+              <p class="govuk-body">Share information about GOV.UK Pay with your broader team. Managing payments
```

Comment:
We need to keep the non-breaking space in here to keep 'GOV.UK Pay' together on one line

Reply:
I've restored these as well as added them where they were missing in other occurrences of `GOV.UK Pay`